unitary/examples/quantum_rpg/npcs.py

```diff
@@ -67,6 +67,14 @@ def npc_action(self, battle, **kwargs) -> str:
         action_choice = random.random()
         return self.act_on_enemy_qubit(enemy_qubit, action_choice, **kwargs)
 
+    def quantopedia_index(self) -> int:
+        """Bit to use for quantopedia.  Should be power of two."""
+        return 1
```

Comment:
Maybe 0 would be a better default? (impossible to learn about it)

Reply:
Done.  Also, good point, I have added docstrings all around and have also added constants for each bit.